cmake/schemes/url_sha1_boost_library.cmake.in

```diff
@@ -192,6 +216,10 @@ else()
   set(link_opts link=static)
 endif()
 
+set(toolset_full_name ${toolset_name})
+if (toolset_version)
```

Comment:
We need to use `string(COMPARE NOTEQUAL "${toolset_version}" "" has_toolset)` to avoid potential problems with [CMP0054](http://www.cmake.org/cmake/help/v3.2/policy/CMP0054.html). E.g. if somebody will set `ndk` variable (in this scheme or toolchain) the build will be broken.